code/_planes+layers.dm

```diff
@@ -75,34 +75,36 @@ What is the naming convention for planes or layers?
 
 #define ABOVE_PLATING_PLANE		-20
 
-	#define CATWALK_LAYER			0
-	#define DISPOSALS_PIPE_LAYER	1
-	#define LATTICE_LAYER			2
-	#define PIPE_LAYER				3
-	#define WIRE_LAYER				4
-	#define VENT_BEZEL_LAYER		5
-	#define WIRE_TERMINAL_LAYER		6
+	#define HOLOMAP_LAYER			1 // NOTE: ENSURE this is equal to the one at ABOVE_TURF_PLANE!
```

Comment:
Yeah because you're using a different macro in that.

If you do the code snippet I put, it'll replace the `TEST` with... `TEST`, so it keeps replacing it with the same thing until it realizes it's in an infinite loop.